pyspark-iris/README.md

```diff
@@ -1,10 +1,13 @@
-# The `pyspark-iris` Kedro starter
+# [ARCHIVED] This starter is no longer maintained.
+You can still find the starter code in older release tags: https://github.com/kedro-org/kedro-starters/tree/0.18.14/pyspark-iris
 
-## Introduction
+## The `pyspark-iris` Kedro starter
+
+### Introduction
```

Comment:
Likewise add the text suggested previously to show how to create the starter with earlier versions of Kedro?